src/components/AnalysisModal.js

```diff
@@ -143,7 +143,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
         backgroundColor: 'white', borderRadius: 5, padding: '1rem', display: 'inline-block', verticalAlign: 'middle', marginBottom: '1rem',
         textAlign: 'center', width: '100%', height: 60
       },
-      image: { verticalAlign: 'middle', height: '100%', width: '40%' },
+      image: { verticalAlign: 'middle', height: '30px' },
```

Comment:
Curious why you preferred an absolute unit (`px`) over relative (`%`)?

Reply:
I'd love your input on this. I played around a little bit with the styles and some of the bounds were erroneous. Height 100% didn't do anything when width was 40% (if I remember correctly)
I forgot to take notes.

I chose to do absolute units because in the other locations the icons were in absolute units.

Reply:
I had to do some reading. What I learnt are:

> Generally speaking, you usually don’t want to control the exact width of an image. Every visitor who comes to your website has a potentially different size screen. If you specify the width, it may be much too small for some users and much too big for others. Most of the time, the best option is to make sure that your image is inside of a responsive (percent-based) container and then let it fill the container.

https://html.com/attributes/img-width

It's good practice to set both `height` and `width` because

> If height and width are set, the space required for an image is reserved when the page is loaded. However, without these attributes, the browser does not know the size of the image, and cannot reserve the appropriate space to it. The effect will be that the page layout will change during loading (while the images load).

https://www.w3schools.com/tags/att_img_height.asp

Those being said, when I applied them to the case above and played around, I didn't see great results so I don't have a better suggestion. 

Nit: could write it as

`height: 30` instead of `height: '30px'`